storage/src/persistent/mod.rs

```diff
@@ -158,12 +157,12 @@ impl PersistentStorage {
     }
 
     #[inline]
-    pub fn context_storage(&self) -> ContextList { self.cs.clone() }
-}
+    pub fn merkle(&self) -> Arc<RwLock<MerkleStorage>> {
+        self.merkle.clone()
+    }
 
-impl Drop for PersistentStorage {
```

Comment:
@michal-stanek 
I dont remember, what was the problem with this drop?
Without that, databases want be flushed in tests or if light-node ends (panic) without ctrl-c, than flush wont be triggered, 
I will wrap it, and all tests worked
```
impl Drop for PersistentStorage {
    fn drop(&mut self) {
        self.flush_dbs();
    }
}
```

Reply:
another problem is that if self.clog.flush() fails, than self.kv.flush() is also not triggered

Reply:
I remember context tests used to crash because of Drop for PersistentStorage in the old implementation. If the tests work now then it's fine.
Yeah we should try flushing both, I'll get rid of expect()

Reply:
Fixed in the new version